src/plugin.ts

```diff
@@ -51,6 +55,28 @@ export default class Plugin {
       usage: config.usage || 0.75,
       function: config.function,
       name: config.name,
+      ...customMetricConfig
+    }
+  }
+
+  customMetricDefaults(customMetric: CustomMetricConfig, functionName: string): CustomMetricConfig {
+    const defaultDimensions: Dimension[] = [
+      {
+        name: "FunctionName",
+        value: functionName,
+      },
+      {
+        name: "Resource",
+        value: `${functionName}:provisioned`,
+      }
+    ]
+
+    return {
+      dimensions: customMetric.dimensions || defaultDimensions,
+      metricName: customMetric.metricName || 'ProvisionedConcurrencyUtilization',
+      namespace: customMetric.namespace || 'AWS/Lambda',
+      statistic: customMetric.statistic || 'Average',
```

Comment:
@fxgtt, thoughts on making this 'Maximum' by default?

Reply:
Yes, that's probably better. Makes the reason for using customMetric more obvious.